crates/cargo-wdk/src/actions/build/error.rs

```diff
@@ -35,19 +35,25 @@ pub enum BuildActionError {
     OneOrMoreRustProjectsFailedToBuild(PathBuf),
     #[error("One or more workspace members failed to build in the workspace: {0}")]
     OneOrMoreWorkspaceMembersFailedToBuild(PathBuf),
+    #[error("Unsupported target arch was detected: {0}")]
+    UnsupportedArchitecture(String),
+    #[error("Failed to detect target arch from `cargo rustc -- --print cfg` output")]
```

Comment:
```suggestion
    #[error("Failed to detect target arch")]
```
Don't expose the internal details of how we detect target arch to the user.

Reply:
Fixed